src/test/java/com/box/sdk/TestConfig.java

```diff
@@ -25,7 +25,7 @@
 
     private TestConfig() { }
 
-    public static void setLogLevel(String levelString) {
+    public static Logger  setLogLevel(String levelString) {
```

Comment:
What about changing the name to `enableLogger(String)`? I think that's a slightly better name for a function that actually returns a logger.